evm_loader/Cargo.toml

```diff
@@ -17,6 +17,7 @@ solana-cli = "=1.17.31"
 solana-cli-config = "=1.17.31"
 solana-client = "=1.17.31"
 solana-program = { version = "=1.17.31", default-features = false }
+solana-program-test = "=1.17.31"
```

Comment:
Where do you use `solana-program-test`? Remove it, if it is not needed.

Reply:
Fixed